modules/ROOT/pages/supported_versions.adoc

```diff
@@ -3,9 +3,10 @@ include::_partials/attributes.adoc[]
 // end {partialsdir}/attributes.adoc[]
 = Supported Versions
 
-Generally speaking, AeroGear Mobile Services aim to support the latest versions of tools and technologies. As of Feb 2019, the following versions are tested and supported:
+Generally speaking, AeroGear Mobile Services aim to support the latest versions of tools and technologies.
+As of July 2019, the following versions are tested and supported:
 
-* Cordova: version 8
-* Nodejs: version 8 and 10
-* Android SDK: API level 27
-* XCode: version 10.1
\ No newline at end of file
+* Cordova: version 9
+* Nodejs: version 10 and 12 LTS
```

Comment:
Version 10 is now LTS. But should we mention it all? Would just `version 10 and 12` be ok?

Reply:
good point. Let's mention both as LTS

Reply:
Food for thought:
Node v**10.0.0** (Current) - https://nodejs.org/fr/blog/release/v10.0.0/
Node v**10.13.0** (**LTS**) - https://nodejs.org/fr/blog/release/v10.13.0/

In terms of accurately presenting info, I think we need to state "Node v10.13.0 (LTS)" if we want to mention LTS as it seems to me that "Node v10" isn't LTS. I'm open to correction on this as I'm not an expert in this area. What are your thoughts on this?

Reply:
We will always work and test on LTS

Reply:
```suggestion
* Nodejs: version Node v10.16.0 (LTS) and Node v12.x
```

Looking at the versions - there's no LTS for Node v12 hence the proposal - open to suggestion.

Reply:
it's explained here https://endoflife.date/nodejs
v12 will become LTS in Oct (as far as I can see)